components/dropdown/dropdown-content-mixin.js

```diff
@@ -197,6 +204,8 @@ export const DropdownContentMixin = superclass => class extends LocalizeCoreElem
 		this.noPaddingFooter = false;
 		this.noPaddingHeader = false;
 		this.noPointer = false;
+		this.mobileBreakpointOverride = 616;
```

Comment:
Setting the default to 616px. If mobile-breakpoint is never specified, it will use the default value.